src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,21 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
```

Comment:
Not all of us are regex experts. Leave a comment in the code with a sample string and what you expect the regex to match. (For testing that you have the correct regex too)